
uint8[50] text
uint8 severity
